templates/test/ci/prow-ci-version/patches/oot-credential-provider-win.yaml

```diff
@@ -0,0 +1,25 @@
+- op: add
+  path: /spec/template/spec/files/-
+  value:
+    content: |
+      $ErrorActionPreference = 'Stop'
+
+      echo "Use OOT credential provider"
+      mkdir C:\var\lib\kubelet\credential-provider
+      curl.exe --retry 10 --retry-delay 5 -L "https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${AZURE_BLOB_CONTAINER_NAME}/${IMAGE_TAG_ACR_CREDENTIAL_PROVIDER}/azure-acr-credential-provider.exe" --output C:\var\lib\kubelet\credential-provider\acr-credential-provider.exe
+      cp C:\var\lib\kubelet\credential-provider\acr-credential-provider.exe C:\var\lib\kubelet\credential-provider\acr-credential-provider
```

Comment:
This is odd. I assume it's because something else needs to refer to it without the `.exe` extension?

Reply:
There's a bug in k/k but so far my fix PR hasn't been merged.
https://github.com/kubernetes/kubernetes/pull/120291